src/HttpClientInterception/HttpRequestInterceptionBuilder.cs

```diff
@@ -97,6 +97,44 @@ public HttpRequestInterceptionBuilder For(Predicate<HttpRequestMessage>? predica
         return this;
     }
 
+    /// <summary>
+    /// Configures the builder to match any request that meets the criteria defined by the specified predicate.
```

Comment:
I think this documentation should be expanded to make it clear that _all_ the predicates need to match (i.e. it's an `&&` not an `||`).